src/init/application/templates/index.html

```diff
@@ -6,8 +6,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="manifest" href="/manifest.json">
-    <script src="/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
-    <link rel="import" href="/src/<%= elementName %>/<%= elementName %>.html">
+    <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
you're right, this isn't a solution. Closing to fix the problem in polymer-build instead!